### ISSUE TITLE
Provide information on how to move ownership

### ISSUE THREAD

**Issue by konobi** [datproject/docs#132]
Users may need to move hosting of a dat instance between machines, including ownership.

Running out of disk space, moving from a playground setup to a more permanent one and handing off to a fellow team member may all be practical reasons for needing to move ownership.

It would be good to have an FAQ entry that provides a set of instructions on which files and/or folders should be moved between instances to do this safely.

**Comment by joehand** [comment]
Yes! I'll grab a few links in the meantime of where this is documented.

**Comment by konobi** [comment]
Thanks. I had hoped to hop on pulling some info into this issue thread, since it's a "common irc question" sort of thing, but have had family emergency pull me away for the meantime.

**Comment by joehand** [comment]
Beaker post about it here: https://medium.com/blue-link-labs/so-you-want-your-decentralized-browser-to-work-correctly-c06c4038ab12
